### PR TITLE
RavenDB-15821 - Cannot Test Credentials when setting up Server-Wide backup

### PR DESCRIPTION
**Pull request by grisha-kotler**
https://issues.hibernatingrhinos.com/issue/RavenDB-15821

### CODE REVIEW

**Comment by ppekrol** [comment]
Does not compile. PR is for 5.1. Issue is for 4.2. Wrong branch?

**Review by ppekrol — APPROVED** [review]

File: src/Raven.Studio/typescript/commands/database/tasks/testPeriodicBackupCredentialsCommand.ts [hunk 1]
```diff
@@ -1,21 +1,19 @@
 import commandBase = require("commands/commandBase");
-import database = require("models/resources/database");
 import endpoints = require("endpoints");
 
 class testPeriodicBackupCredentialsCommand extends commandBase {
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/database/tasks/testPeriodicBackupCredentialsCommand.ts:4]: should we move this command outside commands/database/tasks? it is no longer db related task.

**Comment by ppekrol** [comment]
@grisha-kotler can you address @ml054 comments?

**Comment by grisha-kotler** [comment]
done

**Comment by ppekrol** [comment]
@grisha-kotler can you check failing test?

File: src/Raven.Server/Web/Studio/StudioTasksHandler.cs [hunk 2]
```diff
@@ -263,6 +265,84 @@ public class OfflineMigrationValidation
             public string ErrorMessage { get; set; }
         }
 
+        [RavenAction("/studio-tasks/periodic-backup/test-credentials", "POST", AuthorizationStatus.ValidUser)]
```
Comment (grisha-kotler) [inline src/Raven.Server/Web/Studio/StudioTasksHandler.cs:268]: this endpoint doesn't have any access to private data (just contacting AWS, Azure... with the provided credentials)

**Review by ml054 — APPROVED** [review]

**Pull request merged**